conn_linux.go

```diff
@@ -15,6 +16,8 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+var errInvalidConfiguration = errors.New("netns cannot be set when disableNSThreadLock is set")
```

Comment:
I think I have typically been prefixing these with "netlink:".

Reply:
Maybe name this `errNetNSDisabled` for clarity.

Reply:
good to keep consistency.